src/resources/parser/glb-parser.js

```diff
@@ -1369,7 +1375,8 @@ const createMaterial = function (gltfMaterial, textures, flipV) {
         "KHR_materials_specular": extensionSpecular,
         "KHR_materials_ior": extensionIor,
         "KHR_materials_transmission": extensionTransmission,
-        "KHR_materials_sheen": extensionSheen
+        "KHR_materials_sheen": extensionSheen,
+        "KHR_materials_emissive_strength": extensionEmissiveStrength
```

Comment:
Wondering if we should order the extensions alphabetically as the list grows. 😄

Reply:
Done 😉

Reply:
Done 😉